src/v2/Apps/Partner/Components/NavigationTabs/index.tsx

```diff
@@ -114,6 +117,12 @@ export const NavigationTabsFragmentContainer = createFragmentContainer(
       fragment NavigationTabs_partner on Partner {
         slug
         displayArtistsSection
+        displayWorksSection
+        filteredWorks: filterArtworksConnection(first: 0) {
```

Comment:
Instead of querying for filtered artworks here, you can instead use [this field](https://github.com/artsy/metaphysics/blob/master/src/schema/v2/partner.ts#L380), which will return the number of published artworks that the partner has.

(This is preferred because it will avoid making an extra call to our API).

Reply:
Fixed